src/components/Atoms/AddTaskButton/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+import FONTFAMILY from "../../../variables/font_family";
+import plus from "../../../assets/svg/plus.svg";
+
+const AddTaskButton = ({ onClick }) => {
+  return (
+    <StyledBtn onClick={onClick}>
+      <HoverCircle></HoverCircle>
+      <Img src={plus} />
+      <Mess>タスクを追加</Mess>
+    </StyledBtn>
+  );
+};
+
+const StyledBtn = styled.button`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  width: 126px;
+  height: 24px;
+  border-radius: 12px;
+  padding: 0;
+  border: none;
+  background-color: transparent;
+`;
+
+const HoverCircle = styled.div`
+  position: absolute;
+  width: 126px;
+  height: 24px;
+  border-radius: 12px;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: transparent;
+  ${StyledBtn}:hover > & {
+    background-color: ${COLOR.GREEN};
+    opacity: 0.2;
+  }
+`;
```

Comment:
```suggestion
const HoverCircle = styled.div`
  position: absolute;
  width: 126px;
  height: 24px;
  border-radius: 12px;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
  background-color: transparent;
`;

const StyledBtn = styled.button`
  display: flex;
  justify-content: space-between;
  align-items: center;
  position: relative;
  width: 126px;
  height: 24px;
  border-radius: 12px;
  padding: 0;
  border: none;
  background-color: transparent;

  &:hover {
    & > ${HoverCircle} {
      background-color: ${COLOR.GREEN};
      opacity: 0.2;
    }
  }
`;
```
下のissueを参考に手元で順番を変えてみたら動いたのでこちらで試してみてください！
https://github.com/styled-components/styled-components/issues/3372#issuecomment-758119914

Reply:
ありがとうございます！助かります！
ちなみになんと検索したら解決方法が出てきたか教えていただけますか？

Reply:
https://styled-components.com/docs/advanced#referring-to-other-components
おそらく見出しの Referring to other components で検索したら出てきた気がします！